### PR TITLE
7903124: Add additional helper methods to TestEnvironment

### PR DESCRIPTION
**Pull request by dbessono**
---------
### Progress
- [x] Change must not contain extraneous whitespace

### Issue
 * [CODETOOLS-7903124](https://bugs.openjdk.java.net/browse/CODETOOLS-7903124): Add additional helper methods to TestEnvironment

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jtharness pull/25/head:pull/25` \
`$ git checkout pull/25`

Update a local copy of the PR: \
`$ git checkout pull/25` \
`$ git pull https://git.openjdk.java.net/jtharness pull/25/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 25`

View PR using the GUI difftool: \
`$ git pr show -t 25`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jtharness/pull/25.diff">https://git.openjdk.java.net/jtharness/pull/25.diff</a>

</details>

### CODE REVIEW

File: src/com/sun/javatest/TestEnvironment.java [hunk 1]
```diff
@@ -700,6 +700,153 @@ public void resetElementsUsed() {
...
+        String[] values;
+
+        try {
+            values = lookup(name);
+        } catch (Fault e) {
+            throw new RuntimeException(e);
+        }
+
+        if (values == null) {
+            throw new NullPointerException("Array of values associated with parameter " + name
```
Comment (lkuskov) [inline src/com/sun/javatest/TestEnvironment.java:755]: I don't know how lookup(name) works thus what happen if the method returns empty arrays

Comment (lkuskov) [reply src/com/sun/javatest/TestEnvironment.java:755]: this fact will be masked by IndexOfBound exception that happens on the next level.

**Comment by lkuskov** [comment]
Actually, I would reduce a number of methods, and thrown exceptions. It looks like some methods are superfluous

**Comment by dbessono** [comment]
> Actually, I would reduce a number of methods, and thrown exceptions. It looks like some methods are superfluous



> Actually, I would reduce a number of methods, and thrown exceptions. It looks like some methods are superfluous

**Comment by dbessono** [comment]
> Actually, I would reduce a number of methods, and thrown exceptions. It looks like some methods are superfluous



> Actually, I would reduce a number of methods, and thrown exceptions. It looks like some methods are superfluous

**Comment by lkuskov** [comment]
LGTM

**Pull request closed without merging**